.github/workflows/zeebe-ci.yml

```diff
@@ -186,6 +186,57 @@ jobs:
           secret_vault_secretId: ${{ secrets.VAULT_SECRET_ID }}
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
+  strace-tests:
```

Comment:
Job name needs to be added to the `needs` list of the `test-summary` to make it wait for this new job and fail the build if the new job fails.